src/UIView/UIView+Constraints.h

```diff
@@ -1,5 +1,15 @@
 @import UIKit;
 
+/**
+ *  When a method returns a dictionary, retrieve the desired contraint by using the following keys:
+ */
+NS_ENUM(NSInteger, IPConstraintKey) {
```

Comment:
I think maybe these should be string keys.  It's a little confusing to have an integer as a subscript in ObjC since it requires wrapping.